cypress/support/commands/login.js

```diff
@@ -0,0 +1,24 @@
+const login = (username, password, OTP = null, secondPassword = null) => {
```

Comment:
use ES6 modules
```js
export const login = ()=> {}
```

If you can change the signature of login to destructure an object it would be easier to use rather than having 4 args.
No need to use null too, undefined is the default and it's falsy too ;)

Reply:
I'd prefer to keep the signature. Most of the login actions should be done with the autoLogin command, which will be called with either 0 or 1 parameter(plan type for paid accounts). The login command is for the edge cases where you need a specific setup on the account.

Reply:
Yes but having a signature with more than 3 args is bad :)  Even if it's private it's not easy to use or to add something if we need to add ssomething later.
->look at old dom APIs ;)